Src/CZICmd/CMakeLists.txt

```diff
@@ -26,23 +26,44 @@ IF(UNIX)
  ENDIF()
 ENDIF(UNIX)
 
-find_package(RapidJSON QUIET)
-IF (NOT RapidJSON_FOUND)
-    message(FATAL_ERROR [=[
-    RapidJSON library was not found, which is required for building. Consider installing
-    like 'sudo apt-get install rapidjson-dev'.
```

Comment:
We should also mention vcpkg in the error message

Reply:
next time, I promise